service/src/test/java/uk/gov/hmcts/reform/fpl/utils/OrderIssuedNotificationTestHelper.java

```diff
@@ -51,6 +53,21 @@ private NotifyAdminOrderIssuedTestHelper() {
         return Maps.difference(expectedParameters, resultData);
     }
 
+    public static MapDifference<String, Object> verifyNotificationSentToRepresentativesWhenOrderIssued(
```

Comment:
Would it be possible to generify this and the above method to increase reusability, quite a bit of duplication between the 2 methods

Reply:
good shout 👍